pkg/roachprod/install/cluster_synced.go

```diff
@@ -1634,8 +1635,10 @@ func tenantCertsTarName(virtualClusterID int) string {
 }
 
 // DistributeCerts will generate and distribute certificates to all the nodes.
-func (c *SyncedCluster) DistributeCerts(ctx context.Context, l *logger.Logger) error {
-	if c.checkForCertificates(ctx, l) {
+func (c *SyncedCluster) DistributeCerts(
```

Comment:
We should add a comment about the new `redistribute` argument. As I understand it, I think it means "did we add any new certs" (implying we want to force redistribution). But spelling it out would be better.

Reply:
Done.